.github/workflow-samples/tagged-release.yml

```
@@ -43,7 +43,7 @@ jobs:
       GEO_E2E_ROLE_ARN: ${{ secrets.GEO_E2E_ROLE_ARN }}
       STORAGE_E2E_ROLE_ARN: ${{ secrets.STORAGE_E2E_ROLE_ARN }}
       IN_APP_MESSAGING_E2E_ROLE_ARN: ${{ secrets.IN_APP_MESSAGING_E2E_ROLE_ARN }}
-      LIVENESS_E2E_ROLE_ARN: ${{ secrets.LIVENESS_E2E_ROLE_ARN }}
+      LIVENESS_E2E_ROLE_ARN_NEW: ${{ secrets.LIVENESS_E2E_ROLE_ARN_NEW }}
```

Discussed with Will, to avoid downtime on our current e2e tests we made a new env var. Can swap it back to the old env variable after this pr pushes

---

FYI, we could've also done
```suggestion
      LIVENESS_E2E_ROLE_ARN: ${{ secrets.LIVENESS_E2E_ROLE_ARN_NEW }}
```
so that we only change the value but not the name of the secret. Not blocking, since we're doing a follow up to swap them back anyway.